am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -863,6 +864,65 @@ public void testReturnResourceAccessListWithUserHavingMultipleRelationshipsOnRoo
             .build());
     }
 
+    @Test
+    public void filterResourceWithWildcardAccessorShouldAccessibleToAnyUser() {
+
+        service.grantExplicitResourceAccess(createGrant(resourceId, accessorId, null, resourceDefinition,
+            createPermissions(rootLevelAttribute, ImmutableSet.of(UPDATE))));
+        service.grantExplicitResourceAccess(createGrant(resourceId, "*", null, resourceDefinition,
+            createPermissions(rootLevelAttribute, ImmutableSet.of(READ)), DEFAULT));
+
+
+        FilteredResourceEnvelope result = filterResourceService.filterResource(
+            "anyUser", ImmutableSet.of(idamRoleWithExplicitAccess), createResource(resourceId, resourceDefinition),
+            null);
+
+
+        assertThat(result).isEqualToComparingFieldByField(FilteredResourceEnvelope.builder()
+            .resource(Resource.builder()
+                .id(resourceId)
+                .definition(resourceDefinition)
+                .data(JsonNodeFactory.instance.objectNode())
+                .build())
+            .access(AccessEnvelope.builder()
+                .permissions(ImmutableMap.of(
+                    JsonPointer.valueOf(rootLevelAttribute), ImmutableSet.of(READ)))
+                .accessType(EXPLICIT)
+                .build())
+            .relationships(Collections.emptySet())
+            .build());
+    }
+
+
+    @Test
+    public void filterResourceWithWildcardAccessor() {
```

Comment:
the method name could be changed to make it more meaningful

Reply:
done